recipes/grenedalf/meta.yaml

```
@@ -0,0 +1,50 @@
+package:
+  name: grenedalf
+  version: "0.5.2"
+
+build:
+  number: 0
+  skip: {{ os.environ['TRAVIS_OS_NAME'] == 'osx' }}
```

Travis is not used here at all. 
```suggestion
  skip: true # [osx]
```
I think you can add back linux-aarch64

---

Ah, thank you, that was what I was looking for! Are these things documented anywhere? I've failed finding information on this :-(

---

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#skipping-builds
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocess-selectors